### DISCUSSION TITLE
Does obs_sceneitem_group_from_source actually exsit?

### DISCUSSION THREAD

**Discussion by GeNTooGeek** [obsproject/obs-studio#8767] [Development Questions] [answered]
This function is documented in the Sphinx documentation, but I see no other reference to it in the source code.

**Comment by sebastian-s-beckmann** [comment, marked as answer]
It longer exists since https://github.com/obsproject/obs-studio/commit/35704c4c044d7f44c55a46c70c627f9d34b83a3f (never got into a release on the first place) and should probably be removed.

**Reply by GeNTooGeek** [reply]
Thnx.
